lua/entities/base_wire_entity.lua

```diff
@@ -195,7 +195,8 @@ if CLIENT then
 
 	-- Custom better version of this base_gmodentity function
 	function ENT:BeingLookedAtByLocalPlayer()
-		local trbool = BaseClass.BeingLookedAtByLocalPlayer(self)
+		local trace = LocalPlayer():GetEyeTrace()
```

Comment:
Put this back to `local trbool = BaseClass.BeingLookedAtByLocalPlayer(self)` please

Reply:
Then a distance check cant be done, but i think if i remember the vanilla function does one already?

Reply:
Yeah it does

Reply:
Why? We're already doing all the necessary checks in this function. The base gmod function found here
<https://github.com/Facepunch/garrysmod/blob/master/garrysmod/gamemodes/sandbox/entities/entities/base_gmodentity.lua#L10>
doesn't do anything we need.
In fact it runs an additional trace for seemingly no reason. Not sure why that is even in there at all.

Reply:
No, it runs a different trace if the player's view entity isn't their own entity.

Reply:
Ah I see. Yeah the base function *was* trash back when I made the custom overlay system. Looks like they've improved it a lot, since then.